Source/CronosPlayUnreal/Private/DefiWalletCoreActor.cpp

```diff
@@ -89,257 +89,358 @@ void ADefiWalletCoreActor::Destroyed() {
 }
 
 void ADefiWalletCoreActor::GetNFTSupply(FString denomid, FString nftowner,
-                                        int64 &output, bool &success,
-                                        FString &output_message)
+                                        FGeneralCronosCallInt64Delegate Out) {
+    AsyncTask(ENamedThreads::AnyHiPriThreadNormalTask, [this, denomid, nftowner,
+                                                        Out]() {
+        try {
+            std::string mygrpcstring = TCHAR_TO_UTF8(*myGrpc);
+            rust::cxxbridge1::Box<GrpcClient> grpc_client =
+                new_grpc_client(mygrpcstring);
 
-{
-    try {
-        std::string mygrpcstring = TCHAR_TO_UTF8(*myGrpc);
-        rust::cxxbridge1::Box<GrpcClient> grpc_client =
-            new_grpc_client(mygrpcstring);
+            int64 output = (int64_t)grpc_client->supply(
```

Comment:
int64_t or int64?

Reply:
int64_t is defined in the standard header `<cstdint>`.
so better for compatibility.
i'll change to int64_t